internal/certdb/certdb.go

```diff
@@ -219,11 +233,11 @@ func (db *CertificateRequestsRepository) UpdateUser(id, password string) (int64,
 	if err != nil {
 		return 0, err
 	}
-	insertId, err := result.LastInsertId()
+	affectedRows, err := result.RowsAffected()
```

Comment:
same deal as above. that's why we only update passwords with id, to ensure uniqueness.